packages/@aws-cdk/aws-stepfunctions-tasks/rosetta/with-lambda.ts-fixture

```diff
@@ -0,0 +1,22 @@
+// Fixture with packages imported, but nothing else
+import { Stack } from '@aws-cdk/core';
+import { Construct } from 'constructs';
+import * as sfn from '@aws-cdk/aws-stepfunctions';
+import * as tasks from '@aws-cdk/aws-stepfunctions-tasks';
+import * as lambda from '@aws-cdk/aws-lambda';
+
+class Fixture extends Stack {
```

Comment:
What do you think about collapsing this and the DDB fixture into the default fixture?

Would that get too confusing? I'm wondering what the best practice could be here.

cc @rix0rrr

Reply:
i'm okay with that.
I was also unsure about what the heuristics should be to determine whether a separate fixture is required. I split them up because I imagined the next `n` integrations would probably bloat the default fixture but maybe I got ahead of where we're at currently.

Reply:
That's not clear to me either, but probably prudent to start with fewer fixtures and then diverge as needed.

Reply:
sure, i'm on board with that. will simplify it down into the default fixture!

Reply:
done